site/docs/tutorial/java.md

```diff
@@ -170,7 +170,7 @@ Let's visualize our sample project's dependencies. First, generate a text
 representation of the dependency graph (run the command at the workspace root):
 
 ```
-bazel query  --nohost_deps --noimplicit_deps 'deps(//:ProjectRunner)' \
```

Comment:
Can you change `'` to `"` instead of removing it? I tested it works on Windows. And it helps when the statement contains spaces.